packages/core/utils/src/makeComputedGetters/index.ts

```diff
@@ -0,0 +1,14 @@
+import {computed} from '@vue/composition-api';
+import wrap from '../wrap';
+
+interface Getters<T> {
+  [key: string]: (param: T, ...args: any) => unknown;
```

Comment:
the return type is not unknown, it's Readonly<unknown> :)

Reply:
Changed to `Readonly<Ref<Readonly<T | any>>>;`